dev/ScrollBar/ScrollBar_themeresources.xaml

```diff
@@ -185,16 +224,32 @@
             <SolidColorBrush x:Key="ScrollBarThumbPressedBorderThemeBrush" Color="#ED555555" />
             <SolidColorBrush x:Key="ScrollBarTrackBackgroundThemeBrush" Color="#59D5D5D5" />
             <SolidColorBrush x:Key="ScrollBarTrackBorderThemeBrush" Color="#59D5D5D5" />
-            <StaticResource x:Key="ScrollBarThumbBackgroundColor" ResourceKey="SystemBaseLowColor" />
-            <StaticResource x:Key="ScrollBarPanningThumbBackgroundColor" ResourceKey="SystemChromeDisabledLowColor" />
-            <x:String x:Key="ScrollBarExpandDuration">00:00:00.1</x:String>
+            <StaticResource x:Key="ScrollBarThumbBackgroundColor" ResourceKey="ControlAAFillColorDefault" />
+            <StaticResource x:Key="ScrollBarPanningThumbBackgroundColor" ResourceKey="ControlAAFillColorDefault" />
+
+            <x:String x:Key="ScrollBarExpandDuration">00:00:00.167</x:String>
```

Comment:
These resources can be moved up a level outside of the Light/Dark/HC themes so it doesn't need to be duplicated 3 times

Reply:
@karenbtlai yup, that was my initial thought too but then in the initial ScrollBar these resources are duplicated and may be somebody will want to change the timings in HC theme only (For example). Very unlikely probably :)
@ranjeshj what do you think?

Reply:
If it is the same value, then we don't need to update it when the theme changes, so it makes sense.